unreleased.rst

```diff
@@ -0,0 +1,16 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Release notes
+#############
+
+This page contains release notes for the SDK.
+
+HEAD — ongoing
+--------------
+
+Release Procedure
+~~~~~~~~~~~~~~~~~
+
+- Fixes to the CI/CD release procedure. 
```

Comment:
This is now duplicated (it's in the 0.13.5 release notes).

Reply:
Thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/1762/commits/4869cadbd96c40f6fbbeff766c91b78546228c51

Reply:
Thanks!